pwa/src/apiService/resources/wooRequests.ts

```diff
@@ -0,0 +1,25 @@
+import { IFiltersContext } from "../../context/filters";
+import { TSendFunction } from "../apiService";
+import { AxiosInstance } from "axios";
+
+export default class Verzoeken {

```

Comment:
I'd opt to call this `OpenWoo`, same goes for the file name. 

Right now, we have:

- `wooRequest.ts`
- `Verzoeken` class name
- `/openWoo` endpoint

Three names, for one resource.